validator/rpc/server.go

```diff
@@ -51,20 +44,17 @@ type Config struct {
 	Router                 *mux.Router
 }
 
-// Server defining a gRPC server for the remote signer API.
+// Server defining a HTTP server for the remote signer API and registering clients
 type Server struct {
 	ctx                       context.Context
 	cancel                    context.CancelFunc
-	host                      string
-	port                      string
-	grpcGatewayHost           string
-	grpcGatewayPort           int
-	listener                  net.Listener
+	host                      string // http host, not grpc
+	port                      int    // http port, not grpc
```

Comment:
If you name the fields `httpHost` and `httpPort`, there will be no need for the comments.